src/composer.ts

```diff
@@ -391,11 +391,12 @@ export class Composer<C extends Context> implements MiddlewareObj<C> {
             const set = cmd.indexOf("@") === -1 ? noAtCommands : atCommands;
             set.add(cmd);
         });
-        return this.on(":entities:bot_command").filter(
+        return this.on([":entities:bot_command", ":caption_entities:bot_command"]).filter(
```

Comment:
```suggestion
        return this.on("::bot_command").filter(
```

Reply:
This suggestion is actually not faster. grammY's internal filter query optimisation makes both things equivalent. It is only shorter.